src/Cooling/test_cooling.cpp

```diff
@@ -39,7 +39,16 @@ template <> struct HydroSystem_Traits<CoolingTest> {
   static constexpr double gamma = 5. / 3.; // default value
   // if true, reconstruct e_int instead of pressure
   static constexpr bool reconstruct_eint = true;
-  static constexpr int nscalars = 0;       // number of passive scalars
+};
+
+template <> struct Physics_Traits<CoolingTest> {
+  static constexpr bool is_hydro_enabled = true;
+  static constexpr bool is_radiation_enabled = false;
+  static constexpr bool is_mhd_enabled = false;
+  static constexpr bool is_primordial_chem_enabled = false;
+  static constexpr bool is_metalicity_enabled = false;
```

Comment:
Can we combine these two options into `is_chemistry_enabled`? We will have several types of chemistry, not just primordial chemistry. The use of passive scalars for a metallicity field should be controlled by a supernova feedback option, but let's leave that for a future PR that adds that physics as well.

Reply:
Sounds good. Done.